docs/CONTRIBUTING.md

```diff
@@ -19,33 +19,128 @@ Please submit pull requests in response to open issues. If you have a bug or fea
 
 ## Environment Setup
 
-**Prerequisites:**
+### Prerequisites:
 
 - [Node.js (npm)](https://nodejs.org/en/download/)
-- [Redis](https://redis.io/download)
+- Redis (2 methods)
+  - Use [Docker and docker-compose](#docker-and-docker-compose-set-up)
+  - Install as a [native application](#using-redis-as-a-native-application)
 
-**Redis Set Up**
-Some helpful guides:
+### Using Redis as a native application
 
-- [Redis for Linux](https://redis.io/download#installation) (recommended but not required)
-- [Redis for Windows Subsystem for Linux](https://anggo.ro/note/installing-redis-in-ubuntu-wsl/) (not recommended by Redis)
-- Redis for Windows - You can use [this](https://github.com/tporadowski/redis/releases) installer to get up and running quickly for now.
-- Redis for MacOS - need instructions
-  An easier solution would be to use Docker.
+#### Linux:
 
-**Setup**
+- Install Redis using your distribution's package manager, for example:
+- Ubuntu based: `sudo apt install redis`
+- Red Hat, Fedora: `sudo dnf install redis`
+
+_Once Redis is installed, you can start it in a terminal by running:_
+
+```
+redis-server
+```
+
+#### Windows:
+
+We are using chocolatey package manager to install Redis on Windows. To get chocolatey, simply follow this [guide](https://chocolatey.org/install) and run the following commands:
```

Comment:
I suggest hyperlinking every first instance of newly-introduced tech. In this case, that would entail replacing the first instance of "chocolatey package manager" in this sentence with [Chocolaty package manager](https://chocolatey.org). I also recommend replacing "To get chocolatey" with "To get Chocolatey", as the software's name is stylized with a capital 'C'.

Reply:
Brilliant idea, will get right on it.